tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -82,10 +82,8 @@ def test_find_admin_user(self):
         @Assert: Admin User is found and has Admin role
 
         """
-        with Session(self.browser) as session:
-            session.nav.go_to_users()
-            self.assertIsNotNone(self.user.search('admin'))
-            self.assertTrue(self.user.admin_role_to_user('admin'))
+        with Session(self.browser):
+            self.assertTrue(self.user.user_admin_role_toggle('admin'))
```

Comment:
We have recent improvement for `admin_role_to_user` method. We changed its name and that bring a regression